test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -338,7 +338,7 @@ private void performFailoverTest() throws InterruptedException {
         //Scale down SSS instances to 2
         segmentStoreInstance.scaleService(2, true);
         //zookeeper will take about 60 seconds to detect that the node has gone down
-        Thread.sleep(60000);
+        Thread.sleep(120000);
```

Comment:
Please create an issue to have a better mechanism that is not based on `Thread.sleep`. This issue is actually a clear example of why having to guess a sleeping time is bad in test cases.